packages/block-library/src/quote/deprecated.js

```diff
@@ -151,15 +233,17 @@ const v1 = {
 	migrate( attributes ) {
 		if ( attributes.style === 2 ) {
 			const { style, ...restAttributes } = attributes;
-			return migrateToQuoteV2( {
-				...restAttributes,
-				className: attributes.className
-					? attributes.className + ' is-style-large'
-					: 'is-style-large',
-			} );
+			return migrateTextAlign(
+				migrateToQuoteV2( {
+					...restAttributes,
+					className: attributes.className
+						? attributes.className + ' is-style-large'
+						: 'is-style-large',
+				} )
+			);
 		}
 
-		return migrateToQuoteV2( attributes );
+		return migrateTextAlign( migrateToQuoteV2( attributes ) );
```

Comment:
As noted earlier, this also needs fixing to accept the array returned by `migrateToQuoteV2`